lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -56,13 +56,22 @@
   @include nx-text-helpers.truncate-ellipsis;
   @include nx-text-helpers.regular;
 
-  border: none;
+  border: 1px solid transparent;
```

Comment:
This adds to the height of the item (it is now 34px instead of 32).  To fix this you can either subtract 1 from the padding or, preferably IMO, use an inset outline instead of a border.

Reply:
> This adds to the height of the item (it is now 34px instead of 32). To fix this you can either subtract 1 from the padding or, preferably IMO, use an inset outline instead of a border.

I liked the inset outline idea but TIL that inset in outline doesn't mean the line is drawn inside; It creates an "inset" 3d effect instead. I subtracted 1px from the padding in this commit. Let me know what your thoughts:
c14e789eedf25680dd3cf41dd75086ec2a3a3dac

Reply:
> I liked the inset outline idea but TIL that inset in outline doesn't mean the line is drawn inside; It creates an "inset" 3d effect instead.

Oh sorry I didn't mean that fugliness, I meant https://github.com/sonatype/sonatype-react-shared-components/blob/master/lib/src/base-styles/_nx-pagination.scss#L32

Reply:
Ah thanks, I updated it here:
ab2756aa36df67e2bd9f331dcf949566b0c9f436